casper/src/main/scala/coop/rchain/casper/util/comm/ApproveBlockProtocol.scala

```diff
@@ -0,0 +1,121 @@
+package coop.rchain.casper.util.comm
```

Comment:
It's occurring to me now that this is not the right package for this and `coop.rchain.casper.genesis` probably makes more sense.

Reply:
Hmm but we run this protocol occasionally as the chain progresses right?

Reply:
Yeah, true. This also gives a mechanism for creating signed checkpoints in the future.